hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/staking/StakingRewardsHandlerImpl.java

```diff
@@ -100,7 +101,10 @@ public Map<AccountID, Long> applyStakingRewards(final FinalizeContext context) {
      * @param writableStore The store to write to for updated values
      */
     public void adjustStakedToMeForAccountStakees(@NonNull final WritableAccountStore writableStore) {
-        final var modifiedAccounts = writableStore.modifiedAccountsInState();
+        // If there is a FROM_ACCOUNT_ or _TO_ACCOUNT stake change scenario, the set of modified
+        // accounts returned by writableStore.modifiedAccountsInState(); so we only iterate through
```

Comment:
> the set of modified accounts returned by writableStore.modifiedAccountsInState();

I expected something to complete this statement, did I misread?

Reply:
Ah fixed, oops.